packages/react-components/react-infolabel-preview/stories/InfoLabel/InfoLabelBestPractices.md

```diff
@@ -0,0 +1,10 @@
+<details>
+<summary>
+Best Practices
+</summary>
+
+### Don't
+
+- Because the Popover isn't always visible, don't include information in the InfoButton that people must know in order to complete the field.
```

Comment:
```suggestion
- Because the Popover isn't always visible, don't include information in the `info` prop that people must know in order to complete the field.
```
Nit, not super important to change since the original text also communicates the intent